benchmark/buffers/buffer-base64-decode-wrapped.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [32],
+});
+
+function main(conf) {
+  const n = +conf.n;
+  const charsPerLine = 76;
+  const linesCount = 8 << 16;
+  const bytesCount = charsPerLine * linesCount / 4 * 3;
+
+  const line = 'abcd'.repeat(charsPerLine / 4) + '\n';
+  const data = line.repeat(linesCount);
+  // eslint-disable-next-line no-unescaped-regexp-dot
+  data.match(/./);  // Flatten the string
+  const buffer = Buffer.allocUnsafe(bytesCount);
+  buffer.write(data, 0, bytesCount, 'base64');
```

Comment:
minor nit but this can be simplified just a bit by doing...

```
const line = 'abcd'.repeat(charsPerLine / 4) + '\n';
const buffer = Buffer.alloc(bytesCount, line);
```